packages/smooth_app/android/app/src/main/AndroidManifest.xml

```diff
@@ -52,7 +52,10 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                
+
+                <!-- Prices -->
+                <data android:scheme="http" android:host="prices.openfoodfacts.org" />
```

Comment:
Maybe you should add, the `.net` variant.
Is-it relevant @teolemon?

Reply:
We do have a preprod, indeed

Reply:
Done.